packages/design-system-react/jest.config.js

```diff
@@ -35,6 +36,9 @@ module.exports = merge(baseConfig, {
       statements: 100,
     },
   },
+  moduleNameMapper: {
+    '\\.svg$': '<rootDir>/__mocks__/svgMock.js',
+  },
```

Comment:
Can you provide some context on this change and why it is needed?

Reply:
Removed